src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -1,19 +1,54 @@
 package com.terraformation.backend.accelerator.db
 
 import com.terraformation.backend.accelerator.model.ExistingParticipantProjectSpeciesModel
+import com.terraformation.backend.accelerator.model.NewParticipantProjectSpeciesModel
 import com.terraformation.backend.accelerator.model.ParticipantProjectSpeciesModel
+import com.terraformation.backend.accelerator.model.toModel
 import com.terraformation.backend.auth.currentUser
+import com.terraformation.backend.customer.model.requirePermissions
 import com.terraformation.backend.db.accelerator.ParticipantProjectSpeciesId
+import com.terraformation.backend.db.accelerator.SubmissionStatus
+import com.terraformation.backend.db.accelerator.tables.daos.ParticipantProjectSpeciesDao
+import com.terraformation.backend.db.accelerator.tables.pojos.ParticipantProjectSpeciesRow
 import com.terraformation.backend.db.accelerator.tables.references.PARTICIPANT_PROJECT_SPECIES
 import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.tables.daos.ProjectsDao
 import jakarta.inject.Named
 import org.jooq.Condition
 import org.jooq.DSLContext
 
 @Named
 class ParticipantProjectSpeciesStore(
     private val dslContext: DSLContext,
+    private val participantProjectSpeciesDao: ParticipantProjectSpeciesDao,
+    private val projectsDao: ProjectsDao,
 ) {
+  fun create(model: NewParticipantProjectSpeciesModel): ExistingParticipantProjectSpeciesModel {
+    requirePermissions { createParticipantProjectSpecies(model.projectId) }
+
+    // Participant project species can only be associated
+    // to projects that are associated to a participant
+    val project = projectsDao.fetchOneById(model.projectId)
+    if (project?.participantId == null) {
+      throw ProjectNotInParticipantException(model.projectId)
+    }
+
+    return dslContext.transactionResult { _ ->
+      val row =
+          ParticipantProjectSpeciesRow(
+              feedback = model.feedback,
+              projectId = model.projectId,
+              rationale = model.rationale,
+              speciesId = model.speciesId,
+              submissionStatusId = model.submissionStatus ?: SubmissionStatus.NotSubmitted,
```

Comment:
It might be better to put the default value on the property in the model class so the field can stay non-nullable. That is, in the model class, declare it as `val submissionStatusId: SubmissionStatus = SubmissionStatus.NotSubmitted`.